lib/rubocop/cop/rspec/capybara/specific_actions.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module Capybara
+        # Checks for there is a more specific actions offered by Capybara.
+        #
+        # @example
+        #
+        #   # bad
+        #   find('a').click
+        #   find('button.cls').click
+        #   find('a', exact_text: 'foo').click
+        #   find('div button').click
+        #
+        #   # good
+        #   click_link
+        #   click_button(class: 'cls')
+        #   click_link(exact_text: 'foo')
+        #   find('div').click_button
+        #
+        class SpecificActions < Base
+          include CapybaraHelp
```

Comment:
I was wondering if we could (like with the CssSelector module) not include it, but instead just call methods on the module (using `module_function`)? My hope is that it would be easier to follow the code that way. What do you think?

Reply:
In the `CapybaraHelp` class, I had defined `def_node_matcher`, so I did not make it a `module_function`. Certainly from a code readability standpoint, `module_function` is better, so we changed it.